website/docusaurus.config.js

```diff
@@ -27,7 +27,7 @@ const config = {
     [
       require.resolve("@cmfcmf/docusaurus-search-local"),
       {
-        indexBlog: false,
+        indexDocs: true, // Index docs in search engine
```

Comment:
I think we could be explicit and keep both fields

Reply:
fixed